jenkins/bootstrap.py

```diff
@@ -770,6 +770,7 @@ def setup_root(call, root, repos, ssh, git_cache, clean):
         os.makedirs(root)
     root_dir = os.path.realpath(root)
     logging.info('Root: %s', root_dir)
+    os.environ['GIT_TRACE'] = '1'
```

Comment:
There are a variety of git commands. I want to set it once here. Note that immediately after this is a for loop, where we checkout `len(repos)` repositories.